ghost/deploy.yaml

```diff
@@ -3,17 +3,36 @@ version: "2.0"
 
 services:
   ghost:
-    image: ghost:4.20.4
+    image: ghost:5.12.3
     expose:
       - port: 2368
         as: 80
         to:
           - global: true
-        # change mysite123.com to yours
-        #accept:
-          #- mysite123.com
-    #env:
-      #- "url=http://mysite123.com"
+        accept:
+          - changeme.com
+    env:
+      - NODE_ENV=production
+      - url=http://changeme.com
+      #Change NODE_ENV=production when using a production database
+      #see https://ghost.org/docs/config/#configuration-options
+      - database__connection__host=mysql
```

Comment:
You don't have to use mysql DB when setting `NODE_ENV` to `production.
You can still use `sqlite3` DB as before.

See "Alternatively you can configure sqlite3:" here https://ghost.org/docs/config/#configuration-options

With Akash that would be just passing these 2 env variables:

```
      - database__client=sqlite3
      - database__connection__filename="content/data/ghost-test.db"
```


I've just tested the migration scenario from ghost 4.20 to 5.12, worked flawlessly:

1. create the test volume called "ghost"

```
docker volume create ghost
```

2. start ghost v4.20 with that volume

```
docker run \
  --rm \
  -ti \
  -p 80:2368 \
  -e url=http://localhost \
  -v ghost:/var/lib/ghost/content \
  ghost:4.20.4 
```

3. register a test account and create some post

http://127.0.0.1/ghost/#/setup/one

4. stop and start ghost v5.12 in Production mode using the same DB

```
docker run \
  --rm \
  -ti \
  -p 80:2368 \
  -e url=http://localhost \
  -v ghost:/var/lib/ghost/content \
  -e NODE_ENV=production \
  -e database__client=sqlite3 \
  -e database__connection__filename="content/data/ghost.db" \
  ghost:5.12.3
```

5. everything is working, new ghost finds the DB

http://127.0.0.1

# Next steps

I love the example with the mysql DB, but it is better to have it as a separate deployment manifest in order to support already existing customers using this SDL.

Please:
- [ ] copy this deployment manifest into `deploy-mysql.yaml` file;
- [ ] switch to back to the sqlite3 using the above two lines only and remove the `db` service;

Reply:
=> https://github.com/ovrclk/awesome-akash/pull/317